cmd/util/cmd/root.go

```diff
@@ -65,6 +66,7 @@ func addCommands() {
 	rootCmd.AddCommand(extract.Cmd)
 	rootCmd.AddCommand(export.Cmd)
 	rootCmd.AddCommand(checkpoint_list_tries.Cmd)
+	rootCmd.AddCommand(checkpoint_trie_stats.Cmd)
```

Comment:
This adds the checkpoint trie stats to the util, which is useful to read the trie root hashes of a checkpoint file without reading the entire checkpoint file.